Utils/Extensions/StringExtensions.swift

```diff
@@ -52,6 +52,31 @@ public extension String {
         return str as! String
     }
 
+    /**
+    Truncates a String in the middle only if it's longer than `maxLength`
+
+      "ABCDEF".truncate(4)
+      // "AB…EF"
+
+    :param: maxLength The maximum length of the String.
+
+    :returns: A truncated String
+    */
+    func truncate(#maxLength: Int) -> String {
```

Comment:
This isn't truncation. Truncation means chopping off the end. This is ellipsizing, so this should be called `ellipsize`. (And in the comment, too.)